browser/ui/webui/ai_chat/ai_chat_ui_page_handler.cc

```diff
@@ -307,7 +307,9 @@ void AIChatUIPageHandler::OnNavigated(AssociatedContentDelegate* delegate) {
   // where it would like to remain associated with the Tab and move away from
   // Conversations of previous navigations. That doens't apply to the standalone
   // UI where it will keep a previous navigation's conversation active.
-  chat_ui_->OnNewDefaultConversation();
+
+  // TODO: This is giving the old content_id, not the new one.
+  chat_ui_->OnNewDefaultConversation(active_chat_tab_helper_->GetContentId());
```

Comment:
does the delegate not have it? or maybe this is fixed now and the TODO can be removed?

Reply:
whoops, I fixed this - it was an ordering error